test/e2e/tests/test_tenant_health.py

```
@@ -0,0 +1,270 @@
+    if rc == 0:
+        return json.loads(stdout).get("items", [])
+    if "the server doesn't have a resource type" in stderr.lower():
+        return None
+    if "not found" in stderr.lower() or "no resources found" in stderr.lower():
+        return []
+    raise RuntimeError(f"Failed to list {kind} with label selector: {stderr.strip()}")
+
+
+def _crd_exists(resource, api_group):
+    """Check if a CRD is registered on the cluster."""
+    stdout, stderr, rc = _run_oc(["api-resources", "--api-group", api_group])
```

Ruff (0.15.13)

[warning] Unpacked variable `stderr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)